### PR TITLE
✨ [Frontend] Plus buttons escalate to latest compatible

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Regardless what service key hardcoded in the [new_studies.json](https://github.com/ITISFoundation/osparc-simcore/blob/master/services/static-webserver/client/source/resource/osparc/new_studies.json) file, the plus button will point to the latest compatible service.

- [x] Show versionDisplay in plus button
- [x] Show versionDisplay in loading node page
- [x] Show versionDisplay in update services page
- [x] Relax list ``dev/services``:
  - [x] Frontend: Chunks of 20 items
  - [x] Backend: Timeout increased to 4x5"

PlusButton

image

## Related issue/s

related to https://github.com/ITISFoundation/osparc-issues/issues/1404
closes https://github.com/ITISFoundation/osparc-simcore/pull/5710

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by pcrespov — COMMENTED** [review]
:+1:

**Review by pcrespov — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]
👍

File: packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ async def _call(
             user_id=user_id,
             limit=limit,
             offset=offset,
-            timeout_s=2 * RPC_REQUEST_DEFAULT_TIMEOUT_S,
+            timeout_s=4 * RPC_REQUEST_DEFAULT_TIMEOUT_S,
```
Comment (GitHK) [inline packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py:56]: minor: just define this as a new constant above in this file and explicitly put the value in.

**Review by GitHK — APPROVED** [review]
👍

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by elisabettai — APPROVED** [review]

File: packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py (hunk 1 above)
Comment (odeimaiz) [reply packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py:56]: This is still experimental, we will eventually try to bring it down to one

File: services/static-webserver/client/source/class/osparc/share/PublishTemplate.js [hunk 2]
```diff
@@ -46,6 +46,8 @@ qx.Class.define("osparc.share.PublishTemplate", {
     __selectedCollabs: null,
 
     __buildLayout: function() {
+      // mark it us template, so that testers can share it with product everyone
```
Comment (jsaq007) [inline services/static-webserver/client/source/class/osparc/share/PublishTemplate.js:49]: Not sure the sentence is correct? Would it be correct to say:
// Mark it as template, to that testers can share it with everyone

**Review by jsaq007 — APPROVED** [review]

**Pull request merged**